R/event.R

```diff
@@ -28,7 +30,7 @@
 #'  
 #' @export
 dyEvent <- function(dygraph, 
-                    date,
+                    x,
```

Comment:
We could use `dyEvent <- function(dygraph, x, label, labelLoc, color, strokePattern, date)` then old code with the `date` argument set by position (2) and set using a named argument would still work. What do you think?